pkg/inngest/function.go

```diff
@@ -122,6 +122,15 @@ func (f Function) ConcurrencyLimit() int {
 	return 0
 }
 
+// GetRetries returns the number of retries for the function.
+func (f Function) GetRetries() int {
+	// This uses retries in steps; a holdover from when steps were used in our DAG based approach.
+	if len(f.Steps) == 0 || f.Steps[0].Retries == nil {
+		return consts.DefaultRetryCount
+	}
+	return *f.Steps[0].Retries
+}
```

Comment:
Related to the comment on `retries -> attempts`, it might be better to encapsulate the `+ 1` inside this function and call it `GetAttempts`, as we can clean up the retries/attempts mess when we add `function.retries` in the config. After that:
- If SDK sends `step.retries.attempts` (legacy), `+ 1`
- If SDK sends `function.retries` (future), do not `+ 1`